sycl/test-e2e/syclcompat/math/math_compare.cpp

```diff
@@ -36,6 +36,7 @@
 // RUN: %{run} %t.out
 
 #include <sycl/half_type.hpp>
+#include <sycl/kernel_bundle.hpp>
```

Comment:
I've checked, and this include shouldn't be needed. Could you please check-again? Otherwise, SYCLcompat changes LGTM